drivers/vulkan/vulkan_context.cpp

```diff
@@ -698,13 +697,12 @@ Error VulkanContext::_create_instance() {
 
 	// Initialize extensions.
 	{
-		Error err = _initialize_extensions();
-		if (err != OK) {
+		if (const Error err = _initialize_extensions(); err != OK) {
 			return err;
 		}
 	}
 
-	CharString cs = ProjectSettings::get_singleton()->get("application/config/name").operator String().utf8();
+	const CharString cs = ProjectSettings::get_singleton()->get("application/config/name").operator String().utf8();
```

Comment:
Was it decided at some point to go that far with const correctness? That would require discussion.
I'm in the group that believe that const in local variables and function arguments is of very little value and makes code harder to read and reason about when writing it.